homeassistant/components/airvisual_pro/__init__.py

```diff
@@ -60,6 +60,12 @@ async def async_get_data() -> dict[str, Any]:
         """Get data from the device."""
         try:
             data = await node.async_get_latest_measurements()
+            # If an outdoor sensor use the history to get its values.
```

Comment:
This comment isn't helpful, as it doesn't explain that a `follow_mode` value of `device` equates to an outdoor sensor.